### ISSUE TITLE
File count limits?

### ISSUE THREAD

**Issue by heathdutton** [elcobvg/laravel-opcache#15]
I noticed that unlike the File cache, there is no folder structure created for the files. Does this mean there may be an IO overhead when caching millions of key/value pairs?

**Comment by elcobvg** [comment]
There are no files, it's an in-memory cache

**Comment by heathdutton** [comment]
Uhm... what? This library works by creating files that the opcache engine can load into memory. Similar to other cache engines used in Symfony. These files look like this:

<img width="903" alt="Screen Shot 2019-07-10 at 9 42 13 PM" src="https://user-images.githubusercontent.com/302215/61015884-9d6c9080-a35b-11e9-87e9-93df66374da7.png">

**Comment by elcobvg** [comment]
You're right. Though as you say, the files are loaded into memory. If you are dealing with millions of key/value pairs which won't all fit in available memory, there might be IO overhead to consider. I have never tested that, though. Creating subfolders might help, but I have very little time to work on any open source stuff at the moment.
